tests/data/dataset_readers/conll2003_test.py

```diff
@@ -0,0 +1,53 @@
+import pytest
+
+from allennlp.data.dataset_readers.conll2003 import Conll2003DatasetReader
+from allennlp.common.util import ensure_list
+from allennlp.common.testing import AllenNlpTestCase
+
+
+class TestConll2003Reader:
+    @pytest.mark.parametrize("lazy", (True, False))
+    @pytest.mark.parametrize("coding_scheme", ("IOB1", "BIOUL"))
+    def test_read_from_file_with_deprecated_parameter(self, lazy, coding_scheme):
```

Comment:
It may be desired that `test_read_from_file_with_deprecated_parameter` and `test_read_from_file` are unified with parametrizing (current code is not DRY).

But I didn't introduce new parameterization because it introduces complexity to code and I simply added these tests. We can simply remove `test_file_with_deprecated_parameter` when `coding_scheme` is removed.